sdk/core/core-amqp/test/errors.spec.ts

```diff
@@ -148,19 +158,24 @@ describe("Errors", function() {
         function() {
           const translatedError = <Errors.MessagingError>Errors.translate(mapping as any);
           if (mapping.code === "ECONNRESET") {
-            translatedError.name.should.equal("ServiceUnavailableError");
+            translatedError.name.should.equal("MessagingError");
```

Comment:
Can we not replace this entire if else block with
```javascript
translatedError.name.should.equal("MessagingError");
translatedError.code!.should.equal(mapping.code);
```

Reply:
I refactored these tests so that the if/else only validates the retryable property.